### ISSUE TITLE
Question - Will Monitor Power Off or just show "No HDMI Connected"?

### ISSUE THREAD

**Issue by reillychase** [paviro/MMM-PIR-Sensor#26]
Hi I am working on building my magic mirror, and I think this plugin is the only module able to power off the display when no one is nearby, correct?

With my particular monitor, its settings are locked and the button to unlock them is broken, so no way to change it to power off.

If this module shuts down HDMI connection from the rpi to my monitor, the monitor will display "No HDMI Connected" screen, does that sound right? Any idea to work around this? Thanks

**Comment by paviro** [comment]
You can use switch the power of by using a relay :)

**Comment by reillychase** [comment]
Good idea, thanks for the suggestion! Messing with electricity scares me a bit, I think I'm going to actually get a Belkin Wemo switch and then I am hoping I can figure out how to modify your script to call a python script which will then turn the monitor power on/off when motion detected or not detected for X amount of time. I will also probably want to modify your script to be able to disable motion sensor sometimes like at night because the mirror is in my bedroom and I dont want it turning on when I wake up in the middle of the night or something ha

**Comment by reillychase** [comment]
Update, I tested my monitor with the '/opt/vc/bin/tvservice' on/off commands and it did work actually! Even though the monitor is old / 2011. So thats great, no need for belkin now

**Comment by tjinks17** [comment]
reillychase - I'd like to make the same change to the code as you that you posted on Apr 8. I'm a hack coder though ... What code exactly did you add to which program?  Thank you!
